firefox-ios/Client/Frontend/Settings/Main/AppSettingsTableViewController.swift

```diff
@@ -393,6 +393,7 @@ class AppSettingsTableViewController: SettingsTableViewController,
             FasterInactiveTabs(settings: self, settingsDelegate: self),
             OpenFiftyTabsDebugOption(settings: self, settingsDelegate: self),
             FirefoxSuggestSettings(settings: self, settingsDelegate: self),
+            PasswordGeneratorSetting(settings: self, settingsDelegate: self, windowUUID: windowUUID)
```

Comment:
It doesn't seem that there were feature flags added here, but if we decide to for this work to add the feature flag on the password generator. I would recommend to add to the Feature Flags section since it can avoid this list from being too long and provides a single point for devs / testers to go to show / hide features. 

@DanielDervishi we discussed in slack that the FF may come after, thanks!